src/nrncvode/occvode.cpp

```diff
@@ -176,7 +176,8 @@ printf("%d Cvode::init_eqn id=%d neq_v_=%d #nonvint=%d #nonvint_extra=%d nvsize=
         if (z.cmlcap_) {
             for (auto& ml: z.cmlcap_->ml) {
                 // support `1 x n` and `n x 1` but not `n x m`
-                assert(z.cmlcap_->ml.size() == 1 || ml.nodecount == 1);
+                // why not? (I'm trying to improve lvardt performance)
```

Comment:
```suggestion
                // commented out to improve lvardt performance
```

Reply:
Thanks. At the time my tests first reached the assert, I did not know why ```// support `1 x n` and `n x 1` but not `n x m` ```. Subsequently it became clear there were only a few places this constraint was applicable and those were easy to remove/fix. So there no longer seems a reason to keep this legacy aspect commented upon.